package.json

```diff
@@ -4,12 +4,12 @@
   "description": "if your image doesn't exist, fallback onto another provided image.",
   "main": "lib/index.js",
   "scripts": {
-    "build": "babel src --out-dir lib",
-    "prepublish": "npm run build",
+    "build": "./node_modules/babel-cli/bin/babel.js src --out-dir lib",
```

Comment:
That's because CircleCI build fails. I cannot reproduce in my local environment. I'm suspecting there's some kind of cache problem.

Failed results:
https://circleci.com/gh/socialtables/react-image-fallback/51
https://circleci.com/gh/socialtables/react-image-fallback/54

Fixed result:
https://circleci.com/gh/socialtables/react-image-fallback/55